src/app/services/manage-people/manage-people.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@angular/core';
+import { IUser } from 'interfaces/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ManagePeopleService {
+  readonly LEGAL_AGE = 18;
```

Comment:
Remove this, you are doing DRY here. You have other service that checks legal age, you should add here via Dependency Injection.